commands/version/command.js

```diff
@@ -168,6 +168,11 @@ exports.builder = (yargs, composed) => {
       requiresArg: true,
       defaultDescription: "v",
     },
+    "git-tag-command": {
+      describe:
+        "Allows users to specify a wrapper command in CD/CI pipelines that have no direct write access.",
```

Comment:
```suggestion
        "Allows users to specify a custom command to be used when applying git tags. For example, this may be useful for providing a wrapper command in CI/CD pipelines that have no direct write access.",
```